lib/web_ui/lib/ui_web/src/ui_web/images.dart

```diff
@@ -44,3 +44,17 @@ FutureOr<ui.Image> createImageFromImageBitmap(JSAny imageSource) {
     imageSource as DomImageBitmap,
   );
 }
+
+/// Creates a [ui.Image] from a valid texture source (for example
+/// HTMLImageElement | HTMLVideoElement | HTMLCanvasElement).
+///
+/// By default, [transferOwnership] specifies that the ownership of the texture
+/// will be not be transferred to the renderer, and a copy of the texture source
+/// will be made. If this is not desired, the ownership of the object can be
+/// transferred to the renderer and the engine will take ownership of the
+/// texture source and consume its contents.
+FutureOr<ui.Image> createImageFromTextureSource(Object object,  { required int width, required int height, bool transferOwnership = false }) {
```

Comment:
I think this should actually take a `JSAny` as a parameter instead of a plain `Object`. There are some subtle differences there on the dart2wasm side and `JSAny` is going to be more correct and less error-prone.

Reply:
or `JSObject`?